platform/ios/src/MGLMapView.h

```diff
@@ -1021,6 +1032,182 @@ IB_DESIGNABLE
  */
 - (void)removeOverlays:(NS_ARRAY_OF(id <MGLOverlay>) *)overlays;
 
+#pragma mark Accessing the Underlying Map Data
+
+/**
+ Returns an array of rendered map features that intersect with a given point.
+ 
+ Each object in the returned array represents a feature rendered by the
+ current style and provides access to attributes specified by the relevant
+ vector tile sources. The returned array only includes features specified in
+ vector tile sources, not raster tile sources or other kinds of sources.
+ 
+ Only visible features are returned. For example, suppose the current style uses
+ the
+ <a href="https://www.mapbox.com/vector-tiles/mapbox-streets/">Mapbox Streets source</a>,
+ but none of its style layers includes features that have the `maki` property
+ set to `bus`. If you pass a point corresponding to the location of a bus stop
+ into this method, the bus stop feature does not appear in the resulting array.
+ On the other hand, if the style does include bus stops, an `MGLFeature` object
+ representing that bus stop is returned and its `featureAttributes` dictionary
+ has the `maki` key set to `bus` (along with other attributes). The dictionary
+ does not indicate how the feature is rendered by the current style.
```

Comment:
This explanation is good and correct. As a thought exercise, I wonder what it'd look like to frame this description more in terms of what the API _does_ rather than what it _doesn't_ do. 

This API returns the set of features that are rendered on the map at a point.